src/components/MenuBars/MenuBars.tsx

```diff
@@ -54,7 +61,7 @@ function MenuBars() {
         {"menu-bars--bottom": document.getElementById("menu-bars")?.classList.contains("menu-bars--bottom")}
       )}
     >
-      <section className="menu user-menu">
+      <section className={classNames("menu", "user-menu", {"menu-animation": animate})} onTransitionEnd={(event) => handleAnimate(event)}>
```

Comment:
onTransitionEnd={() => setAnimate(false)} does the same as handleAnimate. Or is the check for event.currentTarget.attributes.getNamedItem("class")?.nodeValue?.includes("menu-animation") necessary?

Reply:
Yes, as mentioned above in the conversation with Johann, this check is needed to react only to events from the component with the class "menu-animation" (`event bubbling`).